swarm/pot/pot_test.go

```diff
@@ -102,23 +101,18 @@ func TestPotRemoveSameBin(t *testing.T) {
 	n := NewPot(newTestAddr("11111111", 0), 0)
 	n, _, _ = testAdd(n, pof, 1, "00000000", "01000000", "01100000", "01110000", "01111000")
 	n, _, _ = Remove(n, newTestAddr("01110000", 0), pof)
-	inds, po := indexes(n)
+	inds := indexes(n)
 	goti := n.Size()
 	expi := 5
 	if goti != expi {
 		t.Fatalf("incorrect number of elements in Pot. Expected %v, got %v", expi, goti)
 	}
-	inds, po = indexes(n)
+	inds = indexes(n)
 	got := fmt.Sprintf("%v", inds)
 	exp := "[5 3 2 1 0]"
 	if got != exp {
 		t.Fatalf("incorrect indexes in iteration over Pot. Expected %v, got %v", exp, got)
 	}
-	got = fmt.Sprintf("%v", po)
```

Comment:
@gluk256 why is this test removed?

Reply:
because i have removed the corresponding variable / return value

Reply:
internal representation of PO does not have any meaning

Reply:
the return value is no longer there